cfg/atmos.cfg

```diff
@@ -1,22 +1,31 @@
 SYMBOLS {
-    __STACKSIZE__: type = weak, value = $0800; # 2k stack
+    __TAPEHDR__:   type = import;
+    __BASHDR__:    type = import;
+    __PROGFLAG__:  type = weak, value = $00; # $00=BASIC, $80=machine code
+    __AUTORUN__:   type = weak, value = $00; # $00=only load, $C7=run
+    __STACKSIZE__: type = weak, value = $0800; # 2K stack
+    __GRAB__:      type = weak, value = 0; # 0=don't grab graphics RAM, 1=grab graphics RAM
+    __RAMEND__:    type = weak, value = $9800 + $1C00 * __GRAB__;
 }
 MEMORY {
     ZP:      file = "", define = yes, start = $00E2, size = $001A;
-    TAPEHDR: file = %O, type   = ro,  start = $0000, size = $0011;
-    RAM:     file = %O, define = yes, start = $0500, size = $9300 - __STACKSIZE__;
+    TAPEHDR: file = %O, type   = ro,  start = $0000, size = $001F;
+    BASHEAD: file = %O, define = yes, start = $0501, size = $000D;
+    RAM:     file = %O, define = yes, start = __BASHEAD_LAST__, size = __RAMEND__ - __RAM_START__ - __STACKSIZE__;
 }
 SEGMENTS {
+    ZEROPAGE: load = ZP,      type = zp;
     TAPEHDR:  load = TAPEHDR, type = ro;
+    BASHDR:   load = BASHEAD, type = ro,  define = yes, optional = yes;
     STARTUP:  load = RAM,     type = ro;
     LOWCODE:  load = RAM,     type = ro,                optional = yes;
-    INIT:     load = RAM,     type = ro,  define = yes, optional = yes;
     CODE:     load = RAM,     type = ro;
     RODATA:   load = RAM,     type = ro;
+    INIT:     load = RAM,     type = ro,  define = yes, optional = yes;
     DATA:     load = RAM,     type = rw;
-    ZPSAVE:   load = RAM,     type = bss, define = yes;
+    ZPSAVE1:  load = RAM,     type = rw,  define = yes;
+    ZPSAVE2:  load = RAM,     type = bss;
```

Comment:
`ZPSAVE1:` and `ZPSAVE2:` are the components of the original `ZPSAVE:` segment.  Together, those two are the same size, sit in the same place, and do the same job, as the original one.

I need to add an extra byte to the end of the program file; but, I don't want to add an _extra_ byte to the program's memory map.  So, I overlay it on the buffer that will save some original content from the zero page.  Therefore, I needed to define two types in the `ZPSAVE:` segment.  But, ld65 won't allow it.  So, I fake it by defining two segments, then using them together as though they're one segment.
[See the "crt0.s" diff.](https://github.com/cc65/cc65/pull/143/files#diff-1f6c7aeb02f4f3f0722799ba433435a8R81)

While the program isn't running, that extra byte protects the program from the ROM bug.  While the program is running, that same space is used by the `zpsave` buffer.